core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,10 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def applyBoost(path: String) =
+    (path startsWith config.rootDir.getAbsolutePath) &&
```

Comment:
this is definitely not guaranteed to work. What you need to do is to check if it is contained within any of the modules roots.